cmake/BuildFlags.cmake

```diff
@@ -123,7 +123,7 @@ if(CMAKE_CXX_COMPILER_ID STREQUAL "GNU" OR CMAKE_CXX_COMPILER_ID STREQUAL "MSVC"
   set(SYCL_OFFLINE_COMPILER_CG_OPTIONS "-options '${SYCL_OFFLINE_COMPILER_CG_OPTIONS}'")
 
   if(WIN32)
-    set(AOT_TARGETS "dg2-g10,mtl-u,mtl-h,xe2-lpg,xe2-hpg")
+    set(AOT_TARGETS "dg2-g10,mtl-u,mtl-h")
```

Comment:
I am concerned with frequent changing of AOT targets without explanation. This might cause different behavior in different pytorch builds and complains from community developers and end users. Besides, how does changing AOT targets relate to the PR subject (issues with restrict)?

What's the reason to drop `xe2-lpg,xe2-hpg` in this commit?

Reply:
@dvrogozh We found another Windows Build error after fixing `restrict` issue. The changes of AOT targets here are just for quick verification of whether the issue is related to binary size. I have removed these changes.